prow/cluster/BUILD

```diff
@@ -1,18 +1,185 @@
+    kind = "Deployment",
+    template = ":deck_deployment.yaml",
+)
+
+k8s_object(
+    name = "deck_service",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "service",
+    template = ":deck_service.yaml",
+)
+
+k8s_object(
+    name = "hook_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":hook_deployment.yaml",
+)
+
+k8s_object(
+    name = "hook_service",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "service",
+    template = ":hook_service.yaml",
+)
+
+k8s_object(
+    name = "horologium_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":horologium_deployment.yaml",
+)
+
+k8s_object(
+    name = "ingress",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "ingress",
+    template = ":ingress.yaml",
+)
+
+k8s_object(
+    name = "jenkins_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":jenkins_deployment.yaml",
+)
+
+k8s_object(
+    name = "jenkins_service",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "service",
+    template = ":jenkins_service.yaml",
+)
+
+k8s_object(
+    name = "lego_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":lego_deployment.yaml",
+)
+
+k8s_object(
+    name = "mem_limit_range",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "limitrange",
+    template = ":mem_limit_range.yaml",
+)
+
+k8s_object(
+    name = "plank_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":plank_deployment.yaml",
+)
+
+k8s_object(
+    name = "prow_job",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "customresourcedfinition",
+    template = ":prow_job.yaml",
+)
+
+k8s_object(
+    name = "push_gateway",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":push_gateway.yaml",
+)
+
+k8s_object(
+    name = "sinker_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":sinker_deployment.yaml",
+)
+
+k8s_object(
+    name = "splice_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":splice_deployment.yaml",
+)
+
+k8s_object(
+    name = "starter",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    # Multiple objects so no kind
+    template = ":starter.yaml",
+)
+
+k8s_object(
+    name = "tide_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":tide_deployment.yaml",
+)
+
+k8s_object(
+    name = "tide_service",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "service",
+    template = ":tide_service.yaml",
+)
+
+k8s_object(
+    name = "tot_deployment",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "deployment",
+    template = ":tot_deployment.yaml",
+)
+
+k8s_object(
+    name = "tot_service",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = "service",
+    template = ":tot_service.yaml",
```

Comment:
FWIW, one of the things I did to cut down on this boilerplate was to adopt naming conventions for files and use lists over `glob()`.  IIRC the naming convention was `name.{kind}.yaml`.

Reply:
Done-ish